src/constants/clickAction.js

```diff
@@ -0,0 +1,5 @@
+// different actions from click events
+export const APPROVE = 'approved';
```

Comment:
One object, many keys for memory. A nit pick indeed but allows us to be conscious of how we are writing code. Also, why not create a global map that uses integers like we did for the society names that we can reuse across the whole code base?

Reply:
@kimani-ndegwa I agree on the single object. But in terms of integer keys, I find this hard to read. Comparing `clickActions[1]` vs clickActions.APPROVE, I would go with the latter. What do you think?